tests/utimensat/01.t

```diff
@@ -0,0 +1,54 @@
+#! /bin/sh
+# $FreeBSD$
+
+desc="utimensat with UTIME_NOW will set the will set typestamps to now"
+
+dir=`dirname $0`
+. ${dir}/../misc.sh
+echo "1..7"
+
+require utimensat
+
+UTIME_NOW=-1
+n0=`namegen`
+n1=`namegen`
+
+expect 0 mkdir ${n1} 0755
+cdir=`pwd`
+cd ${n1}
+
+
+create_file regular ${n0}
+old_mtime=`$fstest lstat ${n0} mtime`
+old_atime=`$fstest lstat ${n0} atime`
+sleep 1	# Ensure that future timestamps will be different than this one
+
+expect 0 open . O_RDONLY : utimensat 0 ${n0} 0 $UTIME_NOW 0 $UTIME_NOW 0
+new_mtime=`$fstest lstat ${n0} mtime`
+new_atime=`$fstest lstat ${n0} atime`
+delta_mtime=$(( $new_mtime - $old_mtime ))
+delta_atime=$(( $new_atime - $old_atime ))
+
+if [ "$delta_mtime" -gt 0 ]; then
+	if [ "$delta_mtime" -lt 300 ]; then
```

Comment:
There's no way for pjdfstest to know exactly what time will get set when it says UTIME_NOW.  So it checks for anything that's "approximately now".  It can't even know what direction the error will be, because ntp or something may come and change the clock.  Five minutes is a pretty arbitrary margin of error.  I'll constantify it.